components/dialog/dialog-styles.js

```diff
@@ -126,7 +126,7 @@ export const dialogStyles = css`
 			padding: 0 20px;
 		}
 		.d2l-dialog-footer {
-			padding: 20px;
+			padding: 20px 20px 2px 20px;
```

Comment:
What happens if for some reason the footer doesn't have any buttons, or are we just not expecting that case?

Reply:
Yeah, so currently we assume there will be at least one button in the footer.  We could add a check for slotted elements and hide the footer completely.  That might be a valid use-case that we haven't seen, or aren't aware of.

Reply:
So... I correct myself... this is actually a use-case we handle for fullscreen dialogs.  We _could_ do a similar check for standard dialogs as well if we think it's worthwhile.

Reply:
This definitely seems better than how it was regardless.

Reply:
Yeah... I think we want to add this check.  We need to decide how we want it to look when it overflows and there's scrolling... i.e. padding-wise and the overflow shadow.  I've put together a couple options and discussing with Jeff to decide what exactly we want.

Reply:
This is the result we decided on... 5px padding for the footer when empty so user can still distinguish the overflow shadow.
![image](https://user-images.githubusercontent.com/9042472/103929487-63fd1880-50eb-11eb-9bcf-bc7070564b03.png)